src/Visualization/Python/GenerateVideo.py

```diff
@@ -0,0 +1,312 @@
+        renderView1.Update()
+        # turn off scalar coloring
+        ColorBy(transform2Display, None)
+        # Hide the scalar bar for this color map if no visible data.
+        HideScalarBarIfNotNeeded(ricciScalarLUT, renderView1)
+        # change solid color
+        transform2Display.AmbientColor = [0.0, 0.0, 0.0]
+        transform2Display.DiffuseColor = [0.0, 0.0, 0.0]
+        # toggle 3D widget visibility (only when running from the GUI)
+        Hide3DWidgets(proxy=transform2.Transform)
+        # set active source
+        SetActiveSource(transform1)
+
+    LoadPalette(paletteName='GradientBackground')
+    # Hide orientation axes
+    renderView1.OrientationAxesVisibility = 0
+    # set active source
+    SetActiveSource(warpByScalar1)
+    # hide color bar/color legend
+    warpByScalar1Display.SetScalarBarVisibility(renderView1, False)
+    # Properties modified on warpByScalar1Display
+    warpByScalar1Display.Opacity = 0.8
+    # get layout
+    layout1 = GetLayout()
+    # layout/tab size in pixels
+    layout1.SetSize(1920, 1080)
+
+    #-----------------------------------
+    # saving camera placements for views
+    # Top down view
+    if (camera_angle == 1):
+        renderView1.CameraPosition = [0.0, 0.0, 36.90869716569761]
+        renderView1.CameraFocalPoint = [0.0, 0.0, 0.6894899550131899]
+        renderView1.CameraViewUp = [0, 1, 0]
+        renderView1.CameraParallelScale = 424.27024700303446
+    # Wide/Inbetween View
+    elif (camera_angle == 2):
+        renderView1.CameraPosition = [
+            -30.093571816984163, -18.55482256667294, 8.558827016411096
+        ]
+        renderView1.CameraFocalPoint = [
+            6.698318283229103e-17, 4.073218385661855e-17, 0.6894899550131903
+        ]
+        renderView1.CameraViewUp = [0.0, 0.0, 1.0]
+        renderView1.CameraParallelScale = 424.27024700303446
+    # Side View
+    else:
+        # current camera placement for renderView1
+        renderView1.CameraPosition = [
+            -29.944619336722987, -3.666072157343372, 2.895224044348878
+        ]
+        renderView1.CameraFocalPoint = [
+            -0.13267040638072278, 0.6356115665206243, -0.37352608789235847
+        ]
+        renderView1.CameraViewUp = [0.0, 0.0, 1.0]
+        renderView1.CameraParallelScale = 519.6152422706632
+
+    # save animation
+    SaveAnimation(output_destination + '/Binary_Pic.png',
+                  renderView1,
+                  ImageResolution=[1920, 1080],
+                  FrameRate=60)
+
+
+@click.command(help=generate_video.__doc__)
+@click.option('--file-destination',
+              '-f',
+              required=True,
+              help="Path to directory containing the volume files xmf and"
+              "h5 files")
+@click.option('--output-destination',
+              '-o',
+              required=True,
+              help="Output directory where pictures will be placed")
+@click.option('--volume-xmf',
```

Comment:
Make this option a `type=click.Path(exists=True, file_ok=True, dir_ok=False, readable=True`, and the aha, ahb options too. Then remove the `file_destination` option because I think you don't need it.